### PR TITLE
SCSS series

### LINKED ISSUE #50: Typo
# Description

in http://localhost:8080/entries/committing-code/

`Commits are like batch-save`

### PR DESCRIPTION
**Pull request by alecmarcus**
## Changes

Adds a new series on the basics of SCSS, with the following entries: 
- CSS Proprocessing (moved from orphan to SCSS Basics)
- SCSS Variables
- Nesting Selectors
- Mixins

## Fixes
Closes #50, #51, #52

### CODE REVIEW

File: src/entries/nesting-selectors.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+.footer__section {
+  padding: 10px;
+}
+
+.footer__heading {
+  font-size: 28px;
+}
+```
+
+Much better. Nesting was not the solution, here. The takeaways are:
```
Comment (stephalee) [inline src/entries/nesting-selectors.md:96]: Love this format 🙌

File: src/entries/nesting-selectors.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+</div>
+
+At the most basic level, nesting rules is a shorthand for adding combinators ([read more](/entries/what-is-css/#descendant-combinator%3A-%5Bspace%5D)) to multiple selectors at once, without having to rewrite the first ingredient. In a more abstract sense, nesting rules allows us to declaratively match the structure of styles to the structure of markup. It also helps reduce repetition when writing code, enables some clever options for composition, and when used smartly, can make styles highly readable and structured.
+
+## Nesting & Specificity
+
+Before we get into it, let's talk about what not to do. It's tempting to nest deeply, and match styles to markup exactly. Doing so can get you into trouble very quickly, as every additional selector adds specificity to a rule. In general, when writing styles (SCSS and CSS), your goal should be to **maintain the minimum level of specificity at all times.**
+
+Here's an inadvisable example of nesting:
```
Comment (stephalee) [inline src/entries/nesting-selectors.md:42]: Maybe this is tacky, but maybe we could include a like 👎 or 🚫 or 🙅 or something to reinforce visually that this is bad practice? Totally just a suggestion!

**Review by stephalee — APPROVED** [review]
This content is awesome!!! I really like the format and style of writing you're starting to establish. It's really parseable!!

**Review by bchiang7 — APPROVED** [review]
Looks great! For some reason when I read these entries I hear them in the Brain Pop guy's voice

image

**Pull request merged**